prem_utils/connectors/openai.py

```diff
@@ -169,20 +169,7 @@ def embeddings(
     ):
         try:
             response = self.client.embeddings.create(model=model, input=input)
-        except (
-            NotFoundError,
-            APIResponseValidationError,
-            ConflictError,
-            APIStatusError,
-            APITimeoutError,
-            RateLimitError,
-            BadRequestError,
-            APIConnectionError,
-            AuthenticationError,
-            InternalServerError,
-            PermissionDeniedError,
-            UnprocessableEntityError,
-        ) as error:
+        except self._errors as error:
```

Comment:
Where is it defined `_errors` variable?

Reply:
```
_errors = except (
            NotFoundError,
            APIResponseValidationError,
            ConflictError,
            APIStatusError,
            APITimeoutError,
            RateLimitError,
            BadRequestError,
            APIConnectionError,
            AuthenticationError,
            InternalServerError,
            PermissionDeniedError,
            UnprocessableEntityError,
        ) as error:
```

Reply:
here: https://github.com/premAI-io/prem-utils/blob/f85ce457f557aab4d0617f4d3dc34d5b4f145f62/prem_utils/connectors/openai.py#L29